cmd/aries-agentd/startcmd/testhelper/testhelper.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package testhelper
+
+import (
+	"errors"
+	"net"
+	"time"
+)
+
+// WaitForServerToStart stalls until the specified host is successfully contacted.
+// An error is returned if no response is received after the given duration is exceeded.
+func WaitForServerToStart(host string, maxTimeToWait time.Duration) error {
```

Comment:
This function is erroneously included in the coverage report because the filename doesn't end with "test"

Reply:
@llorllale Some background info: That WaitForServerToStart method used to be in start_test, but I wanted to use it in another package as well so I moved it to a common place. Doing that caused it to be a part of code coverage, so I added a unit test to it to help mitigate that. Unfortunately it seems like the go compiler won't let me import a file that ends in "test". Is there another solution to this problem? The only other thing I can think of would be to duplicate the WaitForServerToStart code in each package where it's used, but that's ugly as well.

Reply:
move testhelper to internal package,
if testhelper is used in unit-tests then it should have been covered in test coverage.

Refer: https://github.com/hyperledger/aries-framework-go/blob/master/pkg/internal/common/logging/modlog/logtestutils.go

Reply:
Talked to @sudeshrshetty, can't make it an internal package since one of the packages using it is not within the same tree rooted at the parent of the internal directory. So instead I am moving it to pkg/restapi/helper